tensorboard/uploader/exporter.py

```diff
@@ -162,8 +180,20 @@ def export(self, read_time=None):
                 else:
                     raise
 
-    def _request_scalar_data(self, experiment_id, read_time):
-        """Yields JSON-serializable blocks of scalar data."""
+    def _request_json_data(self, experiment_id, read_time):
+        """Given experiment id, generates JSON data and destination file name.
+
+        The JSON data describes the run, tag, metadata, in addition to
+          - Actual data in the case of scalars
+          - Pointer to binary files in the case of blob sequences.
+
+        Args:
+          experiment_id: The id of the experiment to request data for.
+          read_time: Timestamp to use snapshot the Spanner database for reads.
```

Comment:
Is this an `int` milliseconds since epoch, `float` seconds since epoch,
`datetime.datetime` naive value in local time, `datetime.datetime` naive
value in UTC, `datetime.datetime` aware value, `timestamp_pb2.Timestamp`
proto, …?

Also, as in the docs for the protos themselves, we generally try to
avoid specifying exactly which storage mechanism is used.

Can resolve both of these by just following the existing precedent for
`read_time` docs in this module.